### PR TITLE
Search Participant

### PR DESCRIPTION
**Pull request by jkeary**
This PR covers the Search Participant Sidebar, its functionality of going from the Queues View to the Results Participant List View.  It also handles the major routing issues.  And of course, tests for the Search Participant Sidebar.

### CODE REVIEW

File: frontend/src/components/ParticipantsList.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                      <Typography>{participant.pp_id} </Typography>
+                    </TableCell>
+                    <TableCell>
+                      <Typography>{participant.first_name}</Typography>
+                    </TableCell>
+                    <TableCell>
+                      <Typography>{participant.last_name}</Typography>
+                    </TableCell>
+                    <TableCell>
+                      <Typography>Gender</Typography>
```
Comment (hhummel) [inline frontend/src/components/ParticipantsList.js]: Why not fill in the other fields: Gender, Race, DOB? They are all available in  the api

File: frontend/src/components/PerDrawer.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ const useStyles = makeStyles(theme => ({
   },
   drawerPaper: {
     width: drawerWidth,
-    background: theme.palette.background.default,
+    /* background: "#F2FCFF", */
   },
   drawerHeader: {
```
Comment (hhummel) [inline frontend/src/components/PerDrawer.js]: Is background gone now?  Should the commented out line be gone?

File: frontend/src/stores/ParticipantStore.js [hunk 3]
```diff
@@ -1,49 +1,60 @@
...
+    this.params.last_name = data
+  }
+
+  // Getters
+  getParticipantsList = () => {
+    return toJS(this.participants)
+  }
+  getParams = () => {
+    return toJS(this.params)
   }
```
Comment (hhummel) [inline frontend/src/stores/ParticipantStore.js]: Why are you converting these to JSON?

File: frontend/src/components/UserSearch.js [hunk 4]
```diff
@@ -27,11 +51,17 @@ const UserSearch = () => {
...
+        <form className="participant-search__form" onSubmit={handleSubmit}>
           <FormGroup className="participant-search__input">
             <FormControl>
               <InputLabel htmlFor="user_id">User ID</InputLabel>
-              <Input id="user_id" name="user_id" value="" required />
+              <Input
+                id="user_id"
+                name="user_id"
+                value={participantsStore.params.userId}
+                onChange={e => participantsStore.setUserIdParam(e.target.value)}
```
Comment (crsmyz) [inline frontend/src/components/UserSearch.js]: This looks good, I'm hoping it fixes the typing/input issue we were having in this component.

File: frontend/src/stores/ParticipantStore.js (hunk 3 above)
Comment (crsmyz) [reply frontend/src/stores/ParticipantStore.js]: This makes sense if we are storing the data as an observable in the store variables and only converting it to JS on a get call.

File: frontend/src/components/PerDrawer.js (hunk 2 above)
Comment (crsmyz) [reply frontend/src/components/PerDrawer.js]: This looks like a leftover style that just needs to be deleted..

File: frontend/src/stores/ParticipantStore.js (hunk 3 above)
Comment (crsmyz) [reply frontend/src/stores/ParticipantStore.js]: It makes sense if the value in the store is an observable there by needing the JS conversion for the component

File: frontend/src/components/ParticipantsList.js (hunk 1 above)
Comment (jkeary) [reply frontend/src/components/ParticipantsList.js]: @hhummel so, this list view isnt really part of this PR.  Its @angelyoung24 work that will be in a separate PR once she has completed her work on her branch.  So its probably not worth commenting on here

Comment (hhummel) [reply frontend/src/components/ParticipantsList.js]: Got it

File: frontend/src/stores/ParticipantStore.js (hunk 3 above)
Comment (hhummel) [reply frontend/src/stores/ParticipantStore.js]: Let's discuss it tonight

File: frontend/src/components/PerDrawer.js (hunk 2 above)
Comment (jkeary) [reply frontend/src/components/PerDrawer.js]: @hhummel @crsmyz I apologize, this PerDrawer file is no longer in use.  I need to just remove it.  Will do so now.

Comment (jkeary) [reply frontend/src/components/PerDrawer.js]: ok removed the PerDrawer file, it is now in the MainLayout file.

File: frontend/src/components/UserSearch.js (hunk 4 above)
Comment (jkeary) [reply frontend/src/components/UserSearch.js]: indeed it does

**Pull request merged**